src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -601,8 +601,6 @@ public abstract partial class TripleDES : System.Security.Cryptography.Symmetric
     {
         protected TripleDES() { }
         public override byte[] Key { get { throw null; } set { } }
-        public override System.Security.Cryptography.KeySizes[] LegalBlockSizes { get { throw null; } }
```

Comment:
@danmosemsft , @weshaggard 

Not sure where the thinking is on the safety of removing existing overrides from the contract.